app/src/database/crud/crud-forms.js

```diff
@@ -69,9 +69,9 @@ const formsQuery = () => {
       const updateQuery = query.update('forms', { formId }, { latest: latest });
       return await conn.tx(db, updateQuery, [formId]);
     },
-    getMyForms: async ({ id }) => {
-      const sqlQuery = 'SELECT id, name FROM forms WHERE id = $1';
-      const { rows } = await conn.tx(db, sqlQuery, [id]);
+    getMyForms: async () => {
```

Comment:
Hi @navins94 what is the id parameter here previously? is it user id? note that we have to revert the user selection in the future, so multiple user can use the same mobile phone